autoreduce_frontend/static/css/base.css

```diff
@@ -82,7 +82,7 @@ footer {
     text-align: center;
     margin-top: 30px;
     color: #555555;
-    margin-bottom: 10px;
+    margin-bottom: 400px;
```

Comment:
```suggestion
    margin-bottom: 200px;
```

Reply:
I wonder if this can be made a bit smaller so the gap at the bottom is less noticeable?

Reply:
Can do. Although I've noticed a couple of pages: Job Queue and Tokens; don't have enough content to fill out an entire page including the margin-bottom